conf/application.conf

```diff
@@ -1,22 +1,18 @@
 # This is the main configuration file for the application.
 # ~~~~~
 
+play.application.loader=configuration.AppLoader
+
 # Secret key
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-play.crypto.secret=${?APPLICATION_SECRET}
-
-play.modules.enabled += "modules.SentryInitialiserModule"
-
-sentry.dsn=${?SENTRY_DSN}
+play.http.secret.key=${?APPLICATION_SECRET}
```

Comment:
By the time of Play v2.8.8, this value is known as `play.http.secret.key`, and I think `play.crypto.secret` is completely ignored.

https://www.playframework.com/documentation/2.8.x/ApplicationSecret